pkg/metrics/metrics.go

```diff
@@ -118,6 +119,14 @@ var (
 		[]string{"endpoint_state"},
 	)
 
+	// ConntrackGC is the number of deleted entries on the conntrack table by
+	// the garbage collector process.
+	ConntrackGC = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "conntrack_gc",
+		Help:      "Number of deleted entries in the conntrack table by the garbage collector process",
+	}, []string{logfields.EndpointID})
+
```

Comment:
Let's also add CounterVec for:

* conntrack_gc_runs_total - ++ whenever a gc is run
* conntrack_gc_incomplete_runs_total - ++ when gcStats.completed == false
* conntrack_gc_key_fallbacks - gcStats.keyFallback

GaugeVec:
* conntrack_gc_size - gcStats.count *IF* gcStats.completed

Timer:
* conntrack_gc_duration = gcStats.finished.Sub(gcStats.started)